diff --git a/node_modules/@privy-io/react-auth/dist/dts/types-C8c3jEOh.d.ts b/node_modules/@privy-io/react-auth/dist/dts/types-C8c3jEOh.d.ts
index 69a0b4d..2e23881 100644
--- a/node_modules/@privy-io/react-auth/dist/dts/types-C8c3jEOh.d.ts
+++ b/node_modules/@privy-io/react-auth/dist/dts/types-C8c3jEOh.d.ts
@@ -1,3 +1,9 @@
+import type { EmbeddedWalletRecoveryOptions as RecoveryMethod } from '@privy-io/js-sdk-core';
+type EmbeddedSVG = import('react').ComponentType<import('react').SVGProps<SVGSVGElement>>;
+declare const SOLANA_CHAINS: readonly ['solana:mainnet', 'solana:devnet', 'solana:testnet'];
+declare const DATA_SUFFIX_PLUGIN_ID: unique symbol;
+declare const WALLET_CREATION_ON_LOGIN_PLUGIN_ID: unique symbol;
+declare const WALLET_CONNECT_WALLET_CLIENT_TYPES: readonly ["metamask","trust","safe","rainbow","uniswap","zerion","argent","spot","omni","cryptocom","blockchain","safepal","bitget_wallet","zengo","1inch","binance","exodus","mew_wallet","alphawallet","keyring_pro","mathwallet","unstoppable","obvious","ambire","internet_money_wallet","coin98","abc_wallet","arculus_wallet","haha","cling_wallet","broearn","copiosa","burrito_wallet","enjin_wallet","plasma_wallet","avacus","bee","pitaka","pltwallet","minerva","kryptogo","prema","slingshot","kriptonio","timeless","secux","bitizen","blocto","okx_wallet","safemoon","rabby_wallet","bybit_wallet","ronin_wallet","haha_wallet"];
 import { createCoinbaseWalletSDK } from '@coinbase/wallet-sdk';
 import { PublicKeyCredentialCreationOptionsJSON } from '@simplewebauthn/browser';
 import { Rpc, SolanaRpcApi, RpcSubscriptions, SolanaRpcSubscriptionsApi } from '@solana/kit';
@@ -2072,7 +2078,7 @@ type AppConfig = {
         emailDomain?: NonNullable<PrivyClientConfig['appearance']>['emailDomain'];
         landingHeader: NonNullable<PrivyClientConfig['appearance']>['landingHeader'];
         loginMessage: NonNullable<PrivyClientConfig['appearance']>['loginMessage'];
-        footerLogo: NonNullable<PrivyClientConfig['appearance']>['footerLogo'];
+        footerLogo: ReactNode;
         palette: PrivyPalette;
         loginGroupPriority: LoginGroupPriority;
         hideDirectWeb2Inputs?: boolean;
@@ -2099,7 +2105,7 @@ type AppConfig = {
         telegram?: boolean;
     };
     disablePlusEmails: AppResponse['disable_plus_emails'];
-    globalDisablePasskeys: NonNullable<PrivyClientConfig['globalDisablePasskeys']>;
+    globalDisablePasskeys: boolean;
     loginMethodsAndOrder?: PrivyClientConfig['loginMethodsAndOrder'];
     legal: {
         termsAndConditionsUrl: AppResponse['terms_and_conditions_url'];
@@ -2165,7 +2171,7 @@ type AppConfig = {
         extendedCalldataDecoding: NonNullable<NonNullable<PrivyClientConfig['embeddedWallets']>['extendedCalldataDecoding']>;
         transactionScanning: {
             enabled: boolean;
-            domain: NonNullable<NonNullable<NonNullable<PrivyClientConfig['embeddedWallets']>['transactionScanning']>['domain']>;
+            domain: string;
         };
     };
     mfa: {
@@ -2183,7 +2189,7 @@ type AppConfig = {
         isLoading: NonNullable<PrivyClientConfig['customAuth']>['isLoading'];
     };
     render: {
-        standalone: NonNullable<PrivyClientConfig['_render']>['standalone'];
+        standalone: boolean;
     };
     loginConfig: {
         telegramAuthConfiguration?: {
diff --git a/node_modules/@privy-io/react-auth/dist/dts/types-C8c3jEOh.d.mts b/node_modules/@privy-io/react-auth/dist/dts/types-C8c3jEOh.d.mts
index 69a0b4d..2e23881 100644
--- a/node_modules/@privy-io/react-auth/dist/dts/types-C8c3jEOh.d.mts
+++ b/node_modules/@privy-io/react-auth/dist/dts/types-C8c3jEOh.d.mts
@@ -1,3 +1,9 @@
+import type { EmbeddedWalletRecoveryOptions as RecoveryMethod } from '@privy-io/js-sdk-core';
+type EmbeddedSVG = import('react').ComponentType<import('react').SVGProps<SVGSVGElement>>;
+declare const SOLANA_CHAINS: readonly ['solana:mainnet', 'solana:devnet', 'solana:testnet'];
+declare const DATA_SUFFIX_PLUGIN_ID: unique symbol;
+declare const WALLET_CREATION_ON_LOGIN_PLUGIN_ID: unique symbol;
+declare const WALLET_CONNECT_WALLET_CLIENT_TYPES: readonly ["metamask","trust","safe","rainbow","uniswap","zerion","argent","spot","omni","cryptocom","blockchain","safepal","bitget_wallet","zengo","1inch","binance","exodus","mew_wallet","alphawallet","keyring_pro","mathwallet","unstoppable","obvious","ambire","internet_money_wallet","coin98","abc_wallet","arculus_wallet","haha","cling_wallet","broearn","copiosa","burrito_wallet","enjin_wallet","plasma_wallet","avacus","bee","pitaka","pltwallet","minerva","kryptogo","prema","slingshot","kriptonio","timeless","secux","bitizen","blocto","okx_wallet","safemoon","rabby_wallet","bybit_wallet","ronin_wallet","haha_wallet"];
 import { createCoinbaseWalletSDK } from '@coinbase/wallet-sdk';
 import { PublicKeyCredentialCreationOptionsJSON } from '@simplewebauthn/browser';
 import { Rpc, SolanaRpcApi, RpcSubscriptions, SolanaRpcSubscriptionsApi } from '@solana/kit';
@@ -2072,7 +2078,7 @@ type AppConfig = {
         emailDomain?: NonNullable<PrivyClientConfig['appearance']>['emailDomain'];
         landingHeader: NonNullable<PrivyClientConfig['appearance']>['landingHeader'];
         loginMessage: NonNullable<PrivyClientConfig['appearance']>['loginMessage'];
-        footerLogo: NonNullable<PrivyClientConfig['appearance']>['footerLogo'];
+        footerLogo: ReactNode;
         palette: PrivyPalette;
         loginGroupPriority: LoginGroupPriority;
         hideDirectWeb2Inputs?: boolean;
@@ -2099,7 +2105,7 @@ type AppConfig = {
         telegram?: boolean;
     };
     disablePlusEmails: AppResponse['disable_plus_emails'];
-    globalDisablePasskeys: NonNullable<PrivyClientConfig['globalDisablePasskeys']>;
+    globalDisablePasskeys: boolean;
     loginMethodsAndOrder?: PrivyClientConfig['loginMethodsAndOrder'];
     legal: {
         termsAndConditionsUrl: AppResponse['terms_and_conditions_url'];
@@ -2165,7 +2171,7 @@ type AppConfig = {
         extendedCalldataDecoding: NonNullable<NonNullable<PrivyClientConfig['embeddedWallets']>['extendedCalldataDecoding']>;
         transactionScanning: {
             enabled: boolean;
-            domain: NonNullable<NonNullable<NonNullable<PrivyClientConfig['embeddedWallets']>['transactionScanning']>['domain']>;
+            domain: string;
         };
     };
     mfa: {
@@ -2183,7 +2189,7 @@ type AppConfig = {
         isLoading: NonNullable<PrivyClientConfig['customAuth']>['isLoading'];
     };
     render: {
-        standalone: NonNullable<PrivyClientConfig['_render']>['standalone'];
+        standalone: boolean;
     };
     loginConfig: {
         telegramAuthConfiguration?: {
